test/kubemark/master-log-dump.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export KUBERNETES_PROVIDER="kubemark"
+export KUBE_CONFIG_FILE="config-default.sh"
```

Comment:
Yup. Otherwise it doesn't work. I guess it's because log-dump.sh sets this to config-test.sh (https://github.com/kubernetes/kubernetes/blob/master/cluster/log-dump.sh#L61)